### ISSUE TITLE
How to define parameters.py, barcode_info_file if samples already demultiplexed?

### ISSUE THREAD

**Issue by marcomeola** [drmaize/C3S-LAA#1]
I have a multiplex run composed of 20 different barcode libraries with all libraries having the same amplicon fragment (same 1st PCR with all the same primers). The sequencing provider sent me the demultiplexed bam files from which I have generated the ccs with the smrtlink-toolkit (ccs: 20 passes + polish).

How do I need to specify in the parameters.py file that I have no h5 files? Does C3S-LAA work with this only ccs or bam files or do I need to get the multiplexed raw bax.h5 file to get C3S-LAA running?

**Comment by ffrancis** [comment]
Yes, you will need to start with the original multiplexed *raw bax.h5 file*
for C3S-LAA.

**Comment by marcomeola** [comment]
I was told the sequencing was performed with Sequel and not RS II, which is why there is no bax.h5 file and the bam files are the only output of the raw data, though. So there is not even a multiplexed bax.h5 file. Guess your pipeline will not work on Sequel data, right?
